Gemfile

```diff
@@ -57,4 +57,6 @@ gem 'devise-jwt'
 gem 'kaminari'
 gem 'rqrcode'
 gem 'cancancan'
```

Comment:
good choice with this gem!

Reply:
Agree!, cancancan gem seems good option to manage roles